src/topology/metric_space/lipschitz.lean

```diff
@@ -20,6 +20,37 @@ begin
   exact hx.comp hf
 end
 
+/-- A Lipschitz function is uniformly continuous -/
+lemma uniform_continuous_of_lipschitz [metric_space α] [metric_space β] {K : ℝ}
+  {f : α → β} (H : ∀x y, dist (f x) (f y) ≤ K * dist x y) : uniform_continuous f :=
```

Comment:
I would prefer to use `lipschitz_with` in the assumptions, instead of the unfolded version.

Reply:
Do you not want to depend on `lipschitz_with` due to the `0 ≤ K` constraint?

Reply:
Precisely. `0 ≤ K` is not needed for the statement, so I give it here without this assumption, and then I deduce below the version in terms of `lipschitz_with`, in `lipschitz_with.to_uniform_continuous` and `lipschitz_with.to_continuous`.